ts/sagas/notifications.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * A saga to manage notifications
+ */
+
+import { BasicResponseType } from "italia-ts-commons/lib/requests";
+import { Alert, Platform } from "react-native";
+import { call, Effect, select, takeLatest } from "redux-saga/effects";
+
+import { Installation } from "../../definitions/backend/Installation";
+import { PlatformEnum } from "../../definitions/backend/Platform";
+import { BackendClient } from "../api/backend";
+import { apiUrlPrefix, environment } from "../config";
+import I18n from "../i18n";
+import { SESSION_INITIALIZE_SUCCESS } from "../store/actions/constants";
+import {
+  InstallationState,
+  notificationsInstallationSelector
+} from "../store/reducers/notifications/installation";
+import { sessionTokenSelector } from "../store/reducers/session";
+
+const notificationsPlatform: PlatformEnum = Platform.select({
+  ios: PlatformEnum.apns,
+  android: PlatformEnum.gcm
+});
+
+function notifyNotificationsInstallationError() {
+  // If the application is running in dev mode show an alert
+  if (environment === "DEV") {
```

Comment:
can we also send something to mixpanel?

Reply:
Yes absolutly. What you think we can send. For now just a message or you want to fill the message with data (the token/uuid)

Reply:
just a message, it's just a way for us to detect potential issues with the logic